ui/src/components/UserProfile/UserProfile.css

```diff
@@ -0,0 +1,104 @@
+body{margin-top:20px;}
+.card-style1 {
+    box-shadow: 0px 0px 10px 0px rgb(89 75 128 / 9%);
+}
+.border-0 {
+    border: 0!important;
+}
+.card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    background-color: #fff;
+    background-clip: border-box;
+    border: 1px solid rgba(0,0,0,.125);
+    border-radius: 0.25rem;
+}
+
+section {
+    padding: 120px 0;
+    overflow: hidden;
+    background: #fff;
+}
+.mb-2-3, .my-2-3 {
+    margin-bottom: 2.3rem;
+}
+
+.section-title {
+    font-weight: 600;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    margin-bottom: 10px;
+    position: relative;
+    display: inline-block;
+}
+.text-primary {
+    color: #ceaa4d !important;
```

Comment:
it's common for bigger applications to define color constants as css variables (https://stackoverflow.com/questions/1875852/how-can-i-define-colors-as-variables-in-css)

Hex codes can be hard to remember, and it's not necessarily clear what color the text is by just looking at the css file. a variable name can be something like "--main-text-yellow" which is more descriptive. Whenever you use a hex code - try to define it as a variable, so the colors can be reused